### PR TITLE
BB-4097: Add setting OPENEDX_ORACLEJDK_URL

### PR DESCRIPTION
**Pull request by arjunsinghy96**
Adds setting OPENEDX_ORACLEJDK_URL which is used
in appserver configuration variables as `oraclejdk_url`

**JIRA Ticket** [BB-4097](https://tasks.opencraft.com/browse/BB-4097)

**Testing Instructions**
TBD

### CODE REVIEW

File: instance/models/mixins/openedx_config.py
Comment (arjunsinghy96) [reply instance/models/mixins/openedx_config.py:467]: @lgp171188 I verified that `juniper` and `koa` releases use the same JDK version and we only support these versions from OCIM. 

> You can just have the base URL as a separate variable and concatenate it with the appropriate JDK file name for each release, no?

This seems easily doable but redundant to me as we have only one JDK version and this is a stop gap temporary solution.

**Review by lgp171188 — COMMENTED** [review]
@arjunsinghy96, can you resolve the merge conflicts in `settings.py` before I approve and merge this PR?

**Review by lgp171188 — APPROVED** [review]
@arjunsinghy96, :+1: One of the 2 integration test shards is failing due to a flaky error unrelated to the changes in this PR. So once all the other shards complete, re-run the failed shard with the `Restart from failed' option to see if it passes.

Please go ahead and merge this PR after all the CI checks are green.

- [x] I tested this and verified that the `oraclejdk_url` is properly set in Ocim.
- [x] I read through the code
- [x] ~~I checked for accessibility issues~~ NA
- [x] Includes documentation in the form of inline comments.

**Pull request merged**
